iothub/device/src/Transport/RetryDelegatingHandler.cs

```diff
@@ -483,6 +483,38 @@ private async Task EnsureOpenedAsync(CancellationToken cancellationToken)
             }
         }
 
+        private async Task EnsureOpenedAsync(TimeoutHelper timeoutHelper)
```

Comment:
Same as others: avoid code duplication by extracting a method.

Reply:
same as above, does not make much sense here for extracting disposal code.